ElfPack/xcl/K550_R8BA024.xcl

```diff
@@ -0,0 +1,42 @@
+-carm
+//K550_R8BA024
+-Z(CODE)START,CODE,DATA_C,INITTAB=45A40000-45A4FFFF
+
+-Z(CODE)PATCH_SWI=4430C914-4430C91B
+-Z(CODE)LIB=45A50000-45A55000
+
+// change keyhandler
+-Z(CODE)PATCH_KEYHANDLER1=4531B5B0-4531B5B7
+-Z(CODE)PATCH_KEYHANDLER2=4531BA0C-4531BA13
+-Z(CODE)PATCH_KEYHANDLER3=4531BB24-4531BB2B
+-Z(CODE)PATCH_KEYHANDLER4=4531B658-4531B65F
+
+-Z(CODE)PATCH_KEYHANDLER1_STACK1=4531B4BA
+-Z(CODE)PATCH_KEYHANDLER1_STACK2=4531B69A
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE1=4531B5F6
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE2=4531B632
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE3=4531B664
+-Z(CODE)PATCH_KEYHANDLER1_CHANGE4=4531B694
+
+// ���������� elf

```

Comment:
Could you replace comments with those from other xcl files?

Reply:
Oh my fault, will change that tonight. Thank you

Reply:
Done

Reply:
Done